packages/neuron-wallet/tests/store/wallet_store.js

```diff
@@ -103,13 +103,52 @@ describe('wallet store', () => {
   })
 
   it('delete wallet', () => {
-    const walletId = walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet1)
     walletStore.saveWallet(wallet2)
-    walletStore.deleteWallet(walletId)
+    walletStore.deleteWallet(wallet1.id)
+    assert.equal(walletStore.getIDList().length, 1)
     try {
-      walletStore.getWallet(walletId)
+      assert.deepEqual(walletStore.getWallet(wallet1.id), wallet1)
     } catch (e) {
-      assert.deepEqual(e, 0)
+      assert.equal(e, 0)
     }
   })
+
+  it('get and set active wallet', () => {
+    walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet2)
+    let result = walletStore.setActiveWallet(wallet1.id)
+    assert.equal(result, true)
+    let activeWallet = walletStore.getActiveWallet()
+    assert.deepEqual(activeWallet, wallet1)
+    result = walletStore.setActiveWallet(wallet2.id)
+    assert.equal(result, true)
+    activeWallet = walletStore.getActiveWallet()
+    assert.deepEqual(activeWallet, wallet2)
+    result = walletStore.setActiveWallet(wallet1.id)
+    assert.equal(result, true)
+  })
+
+  it('first wallet is active wallet', () => {
+    walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet2)
+    const activeWallet = walletStore.getActiveWallet()
+    assert.deepEqual(activeWallet, wallet1)
+  })
+
+  it('delete active wallet', () => {
+    walletStore.saveWallet(wallet1)
+    walletStore.saveWallet(wallet2)
+    walletStore.deleteWallet(wallet1.id)
+    const activeWallet = walletStore.getActiveWallet()
+    assert.deepEqual(activeWallet, wallet2)
+  })
+
+  it('delte no active wallet', () => {
```

Comment:
`delete` not `delte`, `inactive` not `no active`

Reply:
ok